tests/integration/suite/actors/reminders/basic.go

```diff
@@ -0,0 +1,137 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or impliei.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package healthz
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	rtv1 "github.com/dapr/dapr/pkg/proto/runtime/v1"
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	prochttp "github.com/dapr/dapr/tests/integration/framework/process/http"
+	"github.com/dapr/dapr/tests/integration/framework/process/placement"
+	"github.com/dapr/dapr/tests/integration/framework/util"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(basic))
+}
+
+// basic tests the basic functionality of actor reminders.
+type basic struct {
+	daprd *daprd.Daprd
+	place *placement.Placement
+
+	methodcalled atomic.Int64
+}
+
+func (b *basic) Setup(t *testing.T) []framework.Option {
+	handler := http.NewServeMux()
+	handler.HandleFunc("/dapr/config", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`{"entities": ["myactortype"]}`))
+	})
+	handler.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	})
+	handler.HandleFunc("/actors/myactortype/myactorid/method/remind/remindermethod", func(w http.ResponseWriter, r *http.Request) {
+		b.methodcalled.Add(1)
+	})
+	handler.HandleFunc("/actors/myactortype/myactorid/method/foo", func(w http.ResponseWriter, r *http.Request) {})
+
+	srv := prochttp.New(t, prochttp.WithHandler(handler))
+	b.place = placement.New(t)
+	b.daprd = daprd.New(t, daprd.WithResourceFiles(`
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: mystore
+spec:
+  type: state.in-memory
+  version: v1
+  metadata:
+  - name: actorStateStore
+    value: true
+`),
+		daprd.WithPlacementAddresses("localhost:"+strconv.Itoa(b.place.Port())),
+		daprd.WithAppPort(srv.Port()),
+	)
+
+	return []framework.Option{
+		framework.WithProcesses(b.place, srv, b.daprd),
+	}
+}
+
+func (b *basic) Run(t *testing.T, ctx context.Context) {
+	b.place.WaitUntilRunning(t, ctx)
+	b.daprd.WaitUntilRunning(t, ctx)
+
+	client := util.HTTPClient(t)
+
+	daprdURL := "http://localhost:" + strconv.Itoa(b.daprd.HTTPPort())
+
+	req, err := http.NewRequest(http.MethodPost, daprdURL+"/v1.0/actors/myactortype/myactorid/method/foo", nil)
+	require.NoError(t, err)
+
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
+		resp, rErr := client.Do(req)
+		require.NoError(c, rErr)
+		assert.NoError(c, resp.Body.Close())
+		assert.Equal(c, http.StatusOK, resp.StatusCode)
+	}, time.Second*10, time.Millisecond*100, "actor not ready in time")
+
+	body := `{"dueTime": "100ms"}`
```

Comment:
Consider setting `dueTime` to `0` so the reminder is always executed right away without concerns